backend/roles/role-managers/active-mixplay-users.js

```diff
@@ -0,0 +1,142 @@
+"use strict";
+
+const { ipcMain } = require("electron");
+const settings = require("../../common/settings-access").settings;
+const logger = require("../../logwrapper");
+const userDatabase = require("../../database/userDatabase");
+const Mixplay = require("../../interactive/mixplay");
+
+// Active user toggle
+let activeUserListStatus = settings.getActiveMixplayUserList() === false ? false : true;
+
+// User Timeout Settings
+let userInactiveTimeSetting = settings.getActiveMixplayUserListTimeout() != null ? settings.getActiveMixplayUserListTimeout() : 10;
```

Comment:
Looks like `settings.getActiveMixplayUserListTimeout()` already does a null check so you don't need to do it here too.